realm/realm-library/src/main/java/io/realm/DynamicRealm.java

```diff
@@ -333,7 +333,8 @@ public DynamicRealm freeze() {
      *
      * @param version the schema version to be set.
      */
-    void setVersion(long version) {
+    // FIXME Review public exposure - Test only can be hidden by extension functions
+    public void setVersion(long version) {
```

Comment:
Having this public feels problematic. What classes need to call this that don't live in `io.realm` ?

Reply:
Yes it is problematic. It was used from `SyncedRealmMigrationTests` Moved `SyncedRealmMigrationTests` to `io.realm` and added an extension method `SyncConfiguration.Builder.testSchema` as that was the only reason for putting it in `io.realm.mongodb.sync`.